_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,153 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each target element:
+
+- is categorized to follow the [flow content model][], and;
+- that it's contents are its [descendants][] in the DOM, and;
+- has at least one node in its contents that is a [palpable content][], which does not have the `hidden` attribute specified.
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [HTML Specification - Content model](https://html.spec.whatwg.org/#concept-element-content-model)
+- [HTML Specification - Flow content](https://html.spec.whatwg.org/#flow-content)
+- [HTML Specification - Palpable content](https://html.spec.whatwg.org/#palpable-content)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `ul` element contains [palpable content][] `li` element.
+
+```html
+<ul>
+	<li>Complete online form</li>
+</ul>
+```
+
+#### Passed Example 2
+
+This `dl` element contains palpable content(s), which are name-value group formed by using `dt` and `dd` elements.
+
+```html
+<dl>
+	<dt>Physics</dt>
+	<dd>8/10</dd>
+</dl>
+```
+
+#### Passed Example 3
+
+This `ol` element contains at least one `li` element, which is a valid [palpable content][].
+
+```html
+<ol>
+	<span role="listitem">Eggs</span>
+	<li>Bread</li>
+	<span role="listitem">Milk</span>
+</ol>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `ul` element contains [palpable content][] `li` element, that is [hidden][].
```

Comment:
The element is `aria-hidden`, not `hidden`.
The expectation only talk about `hidden`, not about being in the accessibility tree (`aria-hidden`).